components/application-service/base/kustomization.yaml

```
@@ -12,4 +12,19 @@ images:
 namespace: application-service
 
 patches:
-  - ./manager_resources_patch.yaml
+- ./manager_resources_patch.yaml
+- patch: |-
+    - op: add
+      path: /metadata/labels
+      value:
+        appstudio.redhat.com/namespacetype: "controller"
```

this will replace the already existing labels because you define that you want to add the whole `labels` section with only one value - one label
```suggestion
    - op: add
      path: /metadata/labels/appstudio.redhat.com~1namespacetype
      value:
        controller
```